q2_feature_table/_summarize/summarize_assets/sample-frequency-detail.html

```diff
@@ -1,35 +1,182 @@
-{% extends "tab-child.html" %}
+{% extends "tabbed.html" %} {% block head %}
+<script src="js/vega.min.js"></script>
+<script src="js/vega-embed.min.js"></script>
+<link rel="stylesheet" type="text/css" href="css/spinkit.css" /> {% endblock %}
+{% block tabcontent %}
 
-{% block head %}
-  <script src="./dist/vendor.bundle.js"></script>
-{% endblock %}
+<!-- template it in using jinja -->
+{% if vega_spec is defined %}
+<div class="row">
+  <div class="col-lg-8">
+    <h3>Some Title</h3>
 
-{% block content %}
-  <div class="row">
-    <div class="form-group col-lg-2 col-md-2 col-sm-12 col-xs-12">
-      <label for="slider-value">Sampling depth:</label>
-      <input
-        id="slider-value"
-        class="form-control"
-        type="number"
-        min="0"
-      >
-      <div>(Zero implies no even sampling.)</div>
-      <br>
-      <input
-        id="slider"
-        type="range"
-        value="0"
-        min="0"
-        max="{{ max_count }}"
-      >
-      <br>
+    <div id="plot">
+      <div id="loading" class="spinner">
+        <div class="rect1"></div>
+        <div class="rect2"></div>
+        <div class="rect3"></div>
+        <div class="rect4"></div>
+        <div class="rect5"></div>
+      </div>
     </div>
-    <div id="histogram" class="col-lg-10 col-md-10 col-sm-12 col-xs-12"></div>
   </div>
+
+  <div class="col-lg-4">
+    <div class="row">
+      <div class="col-lg-12"><h3>Plot Controls</h3></div>
+    </div>
+
+    <div class="row"><div class="col-lg-12" id="toolbar"></div></div>
+
+    <div class="row">
+      <div class="col-lg-12" id="metadata-category">Metadata Category</div>
+    </div>
+
+    <div class="row">
+      <div class="col-lg-12" id="sampling-depth-slider"></div>
+      </div>
+
+
+      <div class="row">
+        <div class="col-lg-12" id="text_field">To be updated text!</div>
```

Comment:
Can you match the id style with your other ids? `text-field`. Also, maybe you should remove the "To be updated text!" comment --- this will show up in the viz in the case that a spec isn't able to render, might be confusing to a user to see that.